react-native-purchases-ui/RNPaywalls.podspec

```diff
@@ -17,6 +17,7 @@ Pod::Spec.new do |spec|
   spec.pod_target_xcconfig = { 'DEFINES_MODULE' => 'YES' }
 
   spec.dependency   "React-Core"
-  spec.dependency   "PurchasesHybridCommon", '8.12.0'
+  spec.dependency   "PurchasesHybridCommon", '9.0.0'
```

Comment:
Same as in the flutter one, I wonder if we could remove this dependency, even if you would still need the base SDK to do most things.

Reply:
Done: #864